http_cache/esi.rst

```diff
@@ -137,9 +137,10 @@ matter), Symfony uses the standard ``render`` helper to configure ESI tags:
     .. code-block:: html+php
 
         <!-- app/Resources/views/static/about.html.php -->
-
+        <?php
         // you can use a controller reference
         use Symfony\Component\HttpKernel\Controller\ControllerReference;
+        ?>
         <?php echo $view['actions']->render(
```

Comment:
In other template code examples, each piece of PHP code is surrounded by open/close tags as @javiereguiluz said? I just did the same and I am not sure about what's the correct practice in this case